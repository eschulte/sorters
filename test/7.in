50384 49152 94598 45069 34709 28075 41004 14862 8268 58155 72379 60325 63673 36390 11362 43942 78637 12491 40124 54376