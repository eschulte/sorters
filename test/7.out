8268 11362 12491 14862 28075 34709 36390 40124 41004 43942 45069 49152 50384 54376 58155 60325 63673 72379 78637 94598
